docs/HowToUse.md

```diff
@@ -0,0 +1,28 @@
+# How to use?
+
+Let's compile an example by using ONE-vscode.
+
+## Install toolchains
```

Comment:
```suggestion
## Install the toolchain
```

Reply:
Because we support only one toolchain installed into our system, `toolchains` may confuse users.